client/ayon_core/tools/push_to_project/models/integrate.py

```diff
@@ -952,6 +952,7 @@ def _make_sure_product_exists(self):
             product_name,
             product_type,
             folder_id,
+            attribs=self._src_product_entity["attrib"]
```

Comment:
I wonder if this would also copy attribute like `ftrackId`, `shotgridId`? Maybe it should be more limited instead of "copy all" > causing possible issues.

Reply:
Ok, copied only `description` and `productGroup`.